### PR TITLE
Update Organization Management Guide with Advanced Search feature details

### PR DESCRIPTION
**Pull request by HasiniSama**
## Purpose
The documentation at [WSO2 Identity Server - Organization Management Guide](https://is.docs.wso2.com/en/latest/guides/organization-management/manage-organizations/) is outdated due to the recent addition of the feature: [Extending Organization Filtering to Metadata Attributes](https://github.com/wso2/product-is/issues/20611).

## Goals

- Update the Organization Management Guide to reflect the new UI changes and add a new section on advanced search filtering.

## Approach

- Update Screenshots: All existing organization list page screenshots have been updated to showcase the new placeholder change.
Screenshot 2024-08-08 at 15 18 40

Screenshot 2024-08-08 at 16 42 06

Screenshot 2024-08-08 at 15 32 02

- Add a Section for Organization Search: A new section has been added to the guide for searching an Organization using the Advance Search feature.

Screenshot 2024-08-08 at 16 49 41

## Related PRs
https://github.com/wso2/identity-apps/pull/6667

## Related Issues
https://github.com/wso2/product-is/issues/20864

### CODE REVIEW

**Comment by AnuradhaSK** [comment]
IS UIs don't have `Learn More` sections under title description. Need to replace the IS doc images

File: en/includes/guides/organization-management/manage-organizations.md [hunk 1]
```diff
@@ -55,11 +55,28 @@ To add additional attributes for your organization:
...
+1. Search by **Name** using the search bar.
+2. Search by **query** using the search bar as shown below.
+
+    e.g., name co Best, attributes.Country eq USA. (Multiple filters can be connected using the **and** operator)
+
+    ![organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/organization-advance-search.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+3. Search using the **Advanced Search Dropdown UI** as shown below.
+
+    ![organizations]({{base_path}}/assets/img/guides/organization/manage-organizations/organization-advance-search-dropdown.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
```
Comment (AnuradhaSK) [inline en/includes/guides/organization-management/manage-organizations.md:73]: Shall we mention only 2 options:
1. Using **Advanced Search Dropdown UI**
2. Typing on the search bar

Multiple filters can be connected using the **and** operator and it's example can be added as a note/info

**Comment by HasiniSama** [comment]
On hold until this is resolved: https://github.com/wso2/product-is/issues/20870

File: en/includes/guides/organization-management/manage-organizations.md (hunk 1 above)
Comment (HasiniSama) [reply en/includes/guides/organization-management/manage-organizations.md]: This is now updated to contain those two options.

**Review by AnuradhaSK — APPROVED** [review]

**Pull request merged**